MAINTAINERS.yml

```diff
@@ -166,9 +166,8 @@ ARM arch:
     - include/zephyr/arch/arm/
     - tests/arch/arm/
     - doc/hardware/arch/arm_cortex_m.rst
-    - boards/arm/qemu_cortex_m3/
-    - boards/arm/qemu_cortex_m0/
-    - soc/arm/*
+    - boards/qemu/qemu_cortex_*/
```

Comment:
this matches `    - boards/qemu/qemu_cortex_a53/` from below, but a53 is aarch64 not arm

Reply:
fixed